assignments/pull_requests.go

```diff
@@ -0,0 +1,157 @@
+package assignments
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	pb "github.com/autograde/quickfeed/ag"
+	"github.com/autograde/quickfeed/database"
+	"github.com/autograde/quickfeed/kit/score"
+	"github.com/autograde/quickfeed/scm"
+)
+
+var (
+	// These are used to track how many times someone has been assigned to review a pull request. They map as follows.
+	// teacherReviewCounter[courseID][userID] = count
+	// groupReviewCounter[groupID][userID] = count
+	teacherReviewCounter = make(map[uint64]map[uint64]int)
+	groupReviewCounter   = make(map[uint64]map[uint64]int)
+)
+
+// CreateFeedbackComment formats a feedback comment to be posted on pull requests.
+// It uses the test results from a student commit to create a table like the one shown below.
+// Only the test scores associated with the supplied task are used to generate this table.
+// Table formatting ref: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables
+//
+//  ## Test results from latest push
```

Comment:
Maybe use a full example instead of the many blank cells. Remove spaces before the two header lines. Add newline after the table.